README.md

```diff
@@ -1,32 +1,31 @@
 # Mensinator🩸
 Private period tracking with custom symptoms. We don't track your data - you do. Period.
 
-## 🫂 Made for friends by friends
-Originally Emma made Mensinator for her friend with goals of better symptom tracking and privacy respect.  
-Soon, many of us joined and are now actively building the Mensinator together.  
-It is friendly, straightforward, and respects privacy.  
-We wanted an app to use, not an app to sell data to random corporations.
-
 ## 💡 Features
-- Period and ovulation tracking and statistics,
-- ***Customizable and colour-coded symptoms***,
-- Cycle prediction with average period calculation,
-- Cycle prediction with average ***luteal phase calculation for irregular cycles***,
-- Data export and import for backups or moving to another device,
-- No tracking. Your data is yours. No data is sent anywhere.
-- No bloat. No useless or complex features. No sign-up. No emails.
+- Period and ovulation tracking and statistics.
+- ***Customizable, colour-coded symptoms***.
+- Cycle prediction with average period calculation.
+- Cycle prediction with average ***luteal phase calculation for irregular cycles***.
+- Data export and import for backups or moving to another device.
+- No tracking. No sign-up. Your data is yours. No data is sent anywhere.
+- No bloat. No useless or complex features. No emails.
+
+## 🫂 Made for friends by friends
+Originally, Emma made Mensinator for a friend who was looking for an app that would track unique period symptoms and respect her privacy.
+Soon, more like-minded individuals joined the project. 
+We all wanted a user-friendly, straigtforward app made for us, not an app that uses our data for profit. We are now actively building Mensinator together. 
 
-## 🔮 Future ideas
-- Whatever you suggest: Discuss ideas with users and developers on **[our Discord](https://discord.gg/tHA2k3bFRN)**. Let us know what'd be good to have.
+## 🔮 Future plans
```

Comment:
Mostly curious, no strong need to fix: I like how 'Future ideas' encourages creativity of the reader. 'Future plans' imply it has been decided already and does not encourage interaction as much. I am guessing 'Future ideas' isn't as smooth for native speakers? Are there other options?

Reply:
I changed this one because "ideas" sounds like "we may or may not do this, it's just an idea" and people might avoid making suggestions because they think they won't be seriously considered. "Plans" sounds like, "if you suggest something, we'll work on it. we have a plan to make changes". Like you're saying, it implies the future is decided - but if they read more the decision is to take suggestions, so hopefully that still encourages participation. 
It's another difference in how serious the tone is. I think I'm changing a lot of your wording to be more "serious" because I want a serious period tracker, heh. If other folks like "ideas" better though, that's fine with me, either is acceptable from an English language perspective.